Submitted By: Tushar Teredesai <tushar@linuxfromscratch.org>
Date: 2004-01-24
Initial Package Version: 1.4.2
Origin: Sent by Hamish Greig (SourceMage)
Description: Prevents building of the debug image, saving approximately 200 MB of disk space.

$LastChangedBy: $
$Date: $

--- Build-sun-j2sdk-1.4.2-standard/j2se/make/common/Release.gmk.orig	2004-01-22 20:42:35.000000000 -0600
+++ Build-sun-j2sdk-1.4.2-standard/j2se/make/common/Release.gmk	2004-01-22 20:43:22.000000000 -0600
@@ -157,7 +157,7 @@
 compare-image-jre compare-image-jdk ::
 	@$(ECHO) ">>>Making "$@" @ `$(DATE)` ..."; 
 
-images:: sanity-images post-sanity-images image-jre image-jdk image-jdk-debug
+images:: sanity-images post-sanity-images image-jre image-jdk
 
 #
 # Sources we ship in the SDK.
